lib/propagate-show.js

```diff
@@ -93,6 +95,20 @@ const propagate = instances => {
      *   Transforms are applied in place
      **/
   const propagateStepConditions = (instance) => {
+    if (seen[instance._id]) {
+      return
+    }
+    seen[instance._id] = true
+    if (instance.mountPoint) {
+      const mountPointInstance = instances[instance.mountPoint]
+      propagateStepConditions(mountPointInstance)
+      if (mountPointInstance.show !== undefined) {
```

Comment:
Just a minor thought, the property value should never be undefined right? But rather, the property may not be present? If so, `Object.prototype.hasOwnProperty` could be nice in terms of semantics

...a very minor point however

Reply:
Go for it